cli/inspector.rs

```diff
@@ -547,6 +546,17 @@ impl DenoInspector {
       };
     }
   }
+
+  pub fn wait_for_session_and_break_on_next_statement(&mut self) {
+    if let Some(session) = self.sessions.get_mut().established.iter_mut().next()
+    {
+      // There already is a fully established debugger client connection.
+      session.break_on_next_statement();
+    } else {
+      // Wait for a debugger client to connect first.
+      self.flags.get_mut().waiting_for_session = true;
+    }
```

Comment:
```suggestion
    let mut session_iter = self.sessions.get_mut().established.iter_mut();

    loop {
      if let Some(session) = session_iter.next() {
        // There already is a fully established debugger client connection.
        eprintln!("already establiesh session");
        session.break_on_next_statement();
        break;
      }  
    }
```

Reply:
(another hack) this would now block and wait until devtools establish connection before executing the module - it breaks execution on the first statement of the left-most; bottom-most dependency, but I can consistently get all sources in devtools.